src/lsp/cobol_ptree/compilation_group.ml

```diff
@@ -46,17 +46,26 @@ and program_level =
       }
   | ProgramPrototype
 
+and program_mode = { prog_is_common : bool;
```

Comment:
Could you reformat as the other records (`{` and `}` alone on their lines), and more importantly, put the comment "only within a nested program" on the right of `prog_is_common`?